Makefile

```diff
@@ -3,7 +3,7 @@ BUILD_HASH = $(shell git rev-parse HEAD)
 LDFLAGS += -X "github.com/mattermost/rtcd/service.buildHash=$(BUILD_HASH)"
 
 ## Check go mod files consistency
-.PHONY: gomod-check
+.PHONY: gomod-check check-style golangci-lint test build clean
```

Comment:
`.PHONY: check-style` is defined at `check-style` task at line 11. Is it possible to remove that or define `.PHONY` per task?

Reply:
Missed that. Rebased and fixed!